json/navigation-conf/us.json

```diff
@@ -10,6 +10,11 @@
       "path": "us-news/us-politics",
       "sections": []
     },
+    {
+      "title": "Elections 2020",
+      "path": "/us-news/us-elections-2020",
```

Comment:
The path seem to start with a `/`, are you sure it should?

Reply:
oops, thanks for spotting that!